apps/prairielearn/src/lib/client/page-context.ts

```diff
@@ -37,15 +33,34 @@ export function getPageContext(resLocals: Record<string, any>): PageContext {
   return PageContext.parse(resLocals);
 }
 
-export interface StudentCourseInstanceContext {
-  course_instance: StudentCourseInstance;
-  course: StudentCourse;
-}
+// Since this data comes from res.locals and not the database, we can make certain guarantees
+// about the data.
```

Comment:
Nit, spell out the "certain guarantees"?

```suggestion
// Since this data comes from res.locals and not the database, we can make certain guarantees
// about the data. Specifically, `short_name` will never be null for non-deleted courses
// and course instances.
```